inst/private/python_ipc_driver.m

```diff
@@ -51,6 +51,8 @@
       which_ipc = 'system';
     end
   end
+  
+  which_ipc = 'pytave';
```

Comment:
`sympref ipc pytave`

but this is fine for our testing.  Maybe slightly nicer to change the default in `sympref` (so you can still switch to others for testing).